.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!-- This comments are hidden when you submit the issue, so you do not need to remove them! -->
+<!-- Please be sure to check out our contributing guidelines, https://github.com/sunpy/sunpy/blob/master/CONTRIBUTING.rst. -->
+<!-- Please be sure to check out our code of conduct, https://github.com/sunpy/sunpy/blob/master/CODE_OF_CONDUCT.rst. -->
+
+<!-- Please just have a quick search on GitHub to see if a similar issue has already been posted. -->
+<!-- If a similar issue is closed, have a quick look to see if you are satisfied by the resolution. -->
+<!-- If not please go ahead and open an issue! -->
+
+### Description
+<!-- Provide a general description of the bug or feature request. -->
+<!-- Remember, that issues can be bug reports, feature requests or even just to report something odd. -->
+
+### Expected behavior
+<!-- What did you expect to happen. -->
+<!-- If you think how the code behaves is off, tell us! -->
+
+### Actual behavior
+<!-- What actually happened. -->
+<!-- Was the output confusing or poorly described? -->
+
+### Steps to Reproduce
+<!-- Ideally a code example could be provided so we can run it ourselves. -->
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+### System Details
+<!-- This is part optional but we do want to know at the SunPy version you are using. -->
```

Comment:
why is this optional? imo the example and this info are the most important points in a bug report.